npm/ipsm/ipsm.go

```diff
@@ -87,7 +87,7 @@ func (ipsMgr *IpsetManager) Exists(listName string, setName string, kind string)
 	return true
 }
 
-// IpSetReferIncOrDec checks if an element exists in setMap/listMap and then increases or decreases tis refer count.
+// IpSetReferIncOrDec checks if an element exists in setMap/listMap and then increases or decreases this referCount.
 func (ipsMgr *IpsetManager) IpSetReferIncOrDec(ipsetName string, kind string, countOperation ReferCountOperation) {
```

Comment:
Do you have plan to extend this function for other types in addition to "list" as well?
In this PR, kind is always "list".

Reply:
Yes in the next work item(refer coutn) we will extend support of this IncDec to SetMap where we will track the use of iptable rules and their references to a given ipset. we will track both kinds of ipsets.